### PR TITLE
Add pseudo-finnish translations to Utility

### PR DESCRIPTION
**Pull request by yuderekyu**
### Summary
* Add pseudo-translations for Finnish.

### Deployment Link
https://terra-framework-deploye-pr-121.herokuapp.com/static/#/site/components/application-utility/index

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx [hunk 1]
```diff
@@ -91,9 +92,22 @@ class UtilityMenuItem extends React.Component {
     }
   }
 
+  handleIsSelectable() {
+    if (this.props.isSelectable) {
+      this.setState({ isSelected: !this.state.isSelected });
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx]: Is this setting the state to deselected?

File: packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx [hunk 2]
```diff
@@ -91,9 +92,22 @@ class UtilityMenuItem extends React.Component {
...
+    event.preventDefault();
+    this.handleIsSelectable();
+    this.props.onChange(event, key);
+  }
+
   wrapOnKeyDown(key, onKeyDown) {
     return ((event) => {
       if (event.nativeEvent.keyCode === Utils.KEY_CODES.ENTER || event.nativeEvent.keyCode === Utils.KEY_CODES.SPACE || event.nativeEvent.keyCode === Utils.KEY_CODES.RIGHT_ARROW) {
+        this.handleIsSelectable();
         this.props.onChange(event, key);
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx]: Both `wrapOnKeyDown` and `handleSelection` call `handleIsSelectable` & `onChange`. Could we move the `handleIsSelectable` call to be within the`onChange`?

Comment (emilyrohrbough) [reply packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx]: WHOOPS. JUST KIDDING 😅 its `this.props.onChange`

File: packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx (hunk 1 above)
Comment (yuderekyu) [reply packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx]: No, it's toggling the `isSelected` state

File: packages/terra-application-utility/translations/fi-FI.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "Terra.application.utility.close": "Close",
+  "Terra.application.utility.back": "Back",
+  "Terra.application.utility.button": "Utility Button",
+  "Terra.applicaiton.utility.menu": "Utility Menu"
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/translations/fi-FI.json]: Type here still

Comment (yuderekyu) [reply packages/terra-application-utility/translations/fi-FI.json]: yep got it

**Pull request merged**